packages/forma-36-website/src/components/Layout.css

```diff
@@ -17,3 +17,212 @@ pre {
   margin: 0;
   white-space: normal;
 }
+
+/* OSANO STYLES BEGIN */
+
+.osano-cm-window {
+  font-family: var(--font-stack-primary);
+  z-index: 101;
+}
+
+/* Widget */
+.osano-cm-widget {
+  display: none;
+}
+
+.osano-cm-link {
+  color: var(--color-primary);
+  text-decoration: none;
+}
+.osano-cm-link:focus {
+  box-shadow: var(--glow-primary);
+}
+.osano-cm-link:hover {
+  text-decoration: underline;
+}
+
+/* Dialog */
+.osano-cm-dialog {
+  box-shadow: none;
+  box-sizing: border-box;
+  border-top: 1px solid var(--color-element-light);
+  background: var(--color-white);
+}
+
+.osano-cm-dialog .osano-cm-dialog__list {
+  margin-top: var(--spacing-m);
+}
+
+.osano-cm-dialog .osano-cm-list-item__toggle {
+  margin-right: var(--spacing-m);
+}
+
+.osano-cm-dialog .osano-cm-button {
+  cursor: pointer;
+  font-weight: normal;
+  text-decoration: none;
+  vertical-align: middle;
+  color: var(--color-text-base);
+  font-size: var(--font-size-m);
+  font-family: var(--font-stack-primary);
+  background-color: var(--color-white);
+  border: 2px solid var(--color-element-mid);
+  border-radius: var(--border-radius-medium);
+  line-height: var(--line-height-default);
+}
+
+.osano-cm-dialog .osano-cm-button.osano-cm-save:focus {
+  box-shadow: var(--glow-muted);
+}
+.osano-cm-dialog .osano-cm-button.osano-cm-save:hover {
+  background-color: var(--color-element-lightest);
```

Comment:
can you use new token values?

Reply:
Good catch! Totally forgot about then!

We need to updated then in the experience-components too... I copied this css from there